hs-bindgen/fixtures/macros/macro_in_fundecl_vs_typedef/Example/FunPtr.hs

```diff
@@ -98,7 +98,7 @@ $(HsBindgen.Runtime.Prelude.addCSource (HsBindgen.Runtime.Prelude.unlines
   , "/* test_macrosmacro_in_fundecl_vs_typ_Example_get_struct_name3_ptr */"
   , "__attribute__ ((const))"
   , "void (*hs_bindgen_680e8a5d673ce9c1 (void)) ("
-  , "  struct4 *arg1,"
+  , "  struct struct4 *arg1,"
```

Comment:
Since we do not change `DeclId` at all anymore, we now mirror struct vs typedef usage in C also in the wrappers that we generate. I think that's fine.